docs/source/development/reference/functions.rst

```diff
@@ -93,10 +96,13 @@ paragraph for more details.
 
     The returned :c:type:`PJ`-pointer should be deallocated with :c:func:`proj_destroy`.
 
-    :param PJ_CONTEXT* ctx: Threading context
-    :param int argc: Count of arguments in :c:data:`argv`
-    :param char** argv: Vector of strings with proj-string parameters, e.g. ``+proj=merc``
-    :returns: :c:type:`PJ*`
+    :param ctx: Threading context.
+    :type ctx: :c:type:`PJ_CONTEXT` *
+    :param argc: Count of arguments in :c:data:`argv`
+    :type argc: `int`
+    :param argv: Vector of strings with proj-string parameters, e.g. ``+proj=merc``
```

Comment:
Wouldn't "array" be more correct for C? Vector sounds more like C++.

Reply:
Fixed